mmv1/third_party/terraform/services/resourcemanager/data_source_google_service_accounts.go

```diff
@@ -73,25 +75,29 @@ func datasourceGoogleServiceAccountsRead(d *schema.ResourceData, meta interface{
 
 	accounts := make([]map[string]interface{}, 0)
 
-	accountList, err := config.NewIamClient(userAgent).Projects.ServiceAccounts.List("projects/" + project).Do()
-	if err != nil {
-		return transport_tpg.HandleNotFoundError(err, d, fmt.Sprintf("Service accounts: %s", project))
-	}
+	request := config.NewIamClient(userAgent).Projects.ServiceAccounts.List("projects/" + project)
+
+	err = request.Pages(context.Background(), func(accountList *iam.ListServiceAccountsResponse) error {
```

Comment:
glad to see that there's a simple way to do pagination! If you're interested you can work on https://github.com/hashicorp/terraform-provider-google/issues/19157

standardizing pagination is something we should really look into with more and more issues being brought up about it.

Reply:
Hi @BBBmau ,

thank you very much!
I'll have a look on it.

Kind regards!